src/plugins/WebIndexPlugin.ts

```diff
@@ -48,7 +49,7 @@ $bundles
         }
 
         let jsTags = bundlePaths.map(bundle =>
-            `<script type="text/javascript" src="${bundle}"></script>`
+            `<script ${async ? 'async' : ''} type="text/javascript" src="${bundle}"></script>`
```

Comment:
OPtions I see, but async is not defined. It should be opts.async 
Sorry if I am wrong, doing the review on my phone :)